profiles/arch/riscv/use.mask

```diff
@@ -15,6 +15,10 @@
 -gtk3
 -introspection
 
+# Jonathan Davies (2021-09-03)
+# No arch support
+hyperscan
```

Comment:
This should be a suricata-specific mask in package.use.mask, global masks of local USE flags are frowned upon. It would be nice to have a more detailed explanation (_i.e._ that Hyperscan only works on CPUs with SSSE3) as well.